src/Valitron/Validator.php

```diff
@@ -725,7 +734,7 @@ protected function validateAlphaNum($field, $value)
      */
     protected function validateSlug($field, $value)
     {
-        if (is_array($value)) {
+        if (!is_string($value)) {
```

Comment:
Would previously approve of `123` (integer), `true` (boolean; but not `false` as it translates to `''`) so maybe:
```suggestion
        if (!is_scalar($value)) {
```

Reply:
I thought of that initially but a slug is supposed to be a string a boolean slug is not valid at most an integer slug could be but I figured it wasn't in the spirit of the validator.

on the other hand I see how this could be a 'breaking' change so I can commit the change if my initial sentiment is too 'breaking'